### ISSUE TITLE
[❓ Question]: Convention for formula names differs from meeting 20231113

### ISSUE THREAD

**Issue by Simone-de-Rijke** [Blueprints-org/blueprints#17]
### Preliminary Checks

- [X] I have read the project documentation.
- [X] I have searched the issue tracker for similar questions.

### What Have You Tried?

Just read through Python file chapter_7_servicability_limit_state.py

### Question Description

Python file chapter_7_servicability_limit_state.py has got a class formule name of `Form7Dot3CoefficientKc` instead of `EQ_7_3_coefficient_Kc `. 

I am unsure if there was a reason in the mean time that it changed. 

Just asking to make sure the `Form7Dot3CoefficientKc` was a deliberate notation. 

### Confirmation

- [X] I have checked that this question does not already exist in the issue tracker or documentation.
- [X] I have read and understood the contribution guidelines.

**Comment by bro-wi** [comment]
Hi Simone,

The change is deliberate.

There are two reasons as far as I know. 
1. The reason why we went from **snake_case** to **PascalCase** is because of the pep8 rules for classes. PEP8 also dictates to not use underscores inside a class name, so we had to use the "Dot" in case of a dot. I can't come up with a better solution myself actually.
2. According to the conventions is maths these are not equations but formulas : https://www.mathswithgraham.org.uk/?p=2873

@egarciamendez : anything to add?
